content/kubermatic/master/cheat_sheets/etcd/backup-and-restore/_index.en.md

```diff
@@ -27,34 +27,26 @@ This can be achieved for a Seed by KPP admins through configuring backup destina
 More info on that here [Etcd Backup Destination Settings]({{< ref "../../../tutorials_howtos/administration/admin_panel/backup_buckets" >}}).
 
 {{% notice note %}}
-The legacy way of enabling the automatic etcd backups through the KubermaticConfiguration is still supported, although deprecated. It's advised to migrate to the Seed backup destinations.
+The legacy way of enabling the automatic etcd backups through the KubermaticConfiguration or Seed.Spec.EtcdBackup is not supported since 2.20. Migration to the Seed backup destinations is required.
 {{% /notice %}}
 
 ### Backup and Delete Containers
 
-The new Backup controller is designed to be a drop-in replacement for the legacy backup controller. To achieve this, the legacy `backupStoreContainer` and `backupCleanupContainer` are supported by the new controller. However, it's best to use the new containers to enable the full functionality of the new controller.
+The backup controller uses 2 default jobs to for storing backups and deleting them. If needed, it is possible to configure custom jobs in the KubermaticConfiguration.
 
- * *`backupStoreContainer`*: it's recommended to update the seed controller configuration to use the new backup container. The new container spec is provided in the `charts` directory shipped with the KKP release:
+ * *`backupStoreContainer`*:
  ```bash
  charts/kubermatic/static/store-container-new.yaml
  ```
```

Comment:
This file does not exist anymore since 2.21. Not sure if we still want to reference it here vs. using the source https://github.com/kubermatic/kubermatic/blob/master/pkg/controller/operator/defaults/defaults.go#L714-L842 ?

Reply:
right. As we allow for custom jobs, i added the default ones so users can see them and decide if they want to change something.